### PR TITLE
Update building-setup-linux.rst 

### PR DESCRIPTION
**Pull request by vishal-raj15**
a note to avoid firewall problem

### CODE REVIEW

File: dev/source/docs/building-setup-linux.rst [hunk 1]
```diff
@@ -24,6 +24,9 @@ Clone ArduPilot repository
 .. include:: git-clone.rst
     :start-after: inclusion-marker-do-not-remove
     :end-before: Cloning with the GitHub GUI (Windows or MAC)
+    
+.. note::in case you get a connection error because of firewalls type command :
```
Comment (andyp1per) [inline dev/source/docs/building-setup-linux.rst]: You could say:
"Some firewalls do not allow ssh access which can cause submodule updates to fail, in this instance you can tell git to unilaterally use https through the following command:"

**Comment by auturgy** [comment]
I thought that the Git:// interface had been deprecated. Is this even an issue?

**Comment by vishal-raj15** [comment]
@auturgy may be it is not an issue but some people may stuck on that , i mean beginners like me

File: dev/source/docs/building-setup-linux.rst (hunk 1 above)
Comment (vishal-raj15) [reply dev/source/docs/building-setup-linux.rst]: ook , thanks!

**Comment by Hwurzburg** [comment]
fixed induced wiki build errors, formatting, and minor wording clarification.
@peterbarker if you approve, it can be merged...

**Comment by Hwurzburg** [comment]
@khancyr Pierre, changed format as suggested and pushed to his branch....but I have reservations about adding the note globally....
 if you follow our setup guide, all other accesses besides submodule updates will be via remote alias and already be explicit for https, correct? thats why he only ran into it when he did the submodule update with his firewall setup...therefore someone following the guide would not know they needed the change until the submodule update, correct? 
if its 100% innocuous in all configurations we can add it to the git setup, if not, should it not stay where it is?

**Comment by khancyr** [comment]
ha, I didn't see that we are asking to git clone with http already.
The issue is that it could appear on all setup (mac, windows, cygwin). I guess we can keep it here until we get somebody that complain about this error.
There is also a git-submodules page that could be a good candidate to hold this tips and get linked from the linux page

**Comment by vishal-raj15** [comment]
> offset the git config command of 4 space

@khancyr sory but ,i dont understand what it means

**Comment by Hwurzburg** [comment]
@vishal-raj15 dont worry about it...I already pushed a fix for it into your repo and it appears in this PR now...

**Comment by Hwurzburg** [comment]
and I just pushed the correct version back into your branch...the format is now re- corrected...this should be merged early next week during our wiki call ...thanks for helping to improve the wiki!

**Comment by vishal-raj15** [comment]
@Hwurzburg  thanks for help

**Comment by rmackay9** [comment]
Merged, thanks!

**Pull request merged**